macosx/Controller.m

```diff
@@ -823,31 +810,6 @@ - (void) applicationWillTerminate: (NSNotification *) notification
     [self updateTorrentHistory];
     [fTableView saveCollapsedGroups];
 
-    //remaining calls the same as dealloc
-    [fInfoController release];
-    [fMessageController release];
-    [fPrefsController release];
-
-    [fStatusBar release];
-    [fFilterBar release];
-
-    [fTorrents release];
-    [fDisplayedTorrents release];
-
-    [fAddWindows release];
-    [fAddingTransfers release];
-
-    [fOverlayWindow release];
-    [fBadger release];
-
-    [fAutoImportedNames release];
-
-    [fPreviewPanel release];
-
-    [fConfigDirectory release];
-
-    [fFileWatcherQueue release];
```

Comment:
We may need to be careful here as removing explicit `release` calls changes the order of destruction. For instance, I'm not sure whether closing the session is safe while `fFileWatcherQueue` (`VDKQueue` instance) is still alive.